### PR TITLE
faq: Add a few questions and answers

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @iamxy PTAL

### CODE REVIEW

File: faq.md [hunk 1]
```diff
@@ -38,6 +38,8 @@ TiDB 遵循 MySQL 的权限管理体系，可以创建用户并授予权限。
 
 在创建用户时，可以使用 MySQL 语法，如 "CREATE USER 'test'@'localhost' identified by '123';”，这样就添加了一个用户名为 test，密码为 123 的用户，这个用户只能从 localhost 登录。
 
+修改用户密码时，可以使用 Set Password 语句，例如给 TiDB 的默认 root 用户增加密码："SET PASSWORD FOR 'root'@'%' = '123';"。
```
Comment (siddontang) [inline faq.md]:
```
`Set Password`
```

File: faq.md [hunk 2]
```diff
@@ -99,3 +101,11 @@ TiDB 支持绝大多数 MySQL 语法，一般不需要修改代码。我们提
...
 
 如果你是开发者并且熟习Go语言，你可以尝试在TiDB项目的根目录运行`make parser; ln -s _vendor/src vendor`，之后就可以使用`go run`, `go test` `go install`等命令，但是并不推荐这种做法。
+
+### 为什么修改了 TiKV/PD 的 toml 配置文件，却没有生效？
+
+如果要使用配置文件，请设置 TiKV/PD 的 --config 参数，TiKV/PD 默认情况下不会读取配置文件。
+
+### 为什么 TiKV 数据目录不见了
+
+TiKV 的 --store 参数默认值为 /tmp/tikv/store，在某些虚拟机中，重启操作系统会删除 /tmp 目录下的数据，推荐通过 --store 参数显式设置 TiKV 数据目录。
```
Comment (siddontang) [inline faq.md]:
```
`/tmp/tikv/store`
```

**Comment by ngaut** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
@queenypingcap

**Pull request merged**